### ISSUE TITLE
start_seconds issue

### ISSUE THREAD

**Issue by johnwheeler** [eoyilmaz/timecode#6]
` print(Timecode('29.97', start_seconds=0))`

yields

`23:59:59;29`

Should it yield

`00:00:00:00`?

**Comment by eoyilmaz** [comment]
No it should not!

if it yields to ``00:00:00:00`` then it means that there is a frame with that Timecode. Check this:

```python
tc = Timecode('29.97', start_seconds=0)
assert tc.frames == 0
```

May be we should not accept 0 for ``start_seconds`` parameter. Because it it should not yield to a result. There is no timecode for a zero length footage.

What do you think?

**Comment by eoyilmaz** [comment]
This is resolved in 7951a215619751d19e8e5f327ceab0e1ab7ade89 .

Passing 0 to ``start_seconds`` argument will now raise a ``ValueError`` to clarify that it creates an ambiguity.
